docs/content/latest/yugabyte-platform/configure-yugabyte-platform/set-up-cloud-provider/kubernetes.md

```diff
@@ -60,15 +60,107 @@ showAsideToc: true
 
 </ul>
 
-This page details how to configure Kubernetes for YugabyteDB universes using the Yugabyte Platform. If no cloud providers are configured in the Yugabyte Platform console yet, the main Dashboard page highlights the need to configure at least one cloud provider.
+This page details how to configure Kubernetes provider for YugabyteDB universes using the Yugabyte Platform. If no cloud providers are configured in the Yugabyte Platform console yet, the main Dashboard page highlights the need to configure at least one cloud provider.
 
 ![Configure Cloud Provider](/images/ee/configure-cloud-provider.png)
 
 ## Prerequisites
 
 ### Kubernetes
 
-If you plan to run YugabyteDB nodes on Kubernetes, all you need to provide in the Yugabyte Platform console is your cloud provider credentials. The Yugabyte Platform will use those credentials to automatically provision and de-provision instances that run Yugabyte. An 'instance' for YugabyteDB includes a compute instance as well as local or remote disk storage attached to the compute instance.
+If you plan to run YugabyteDB nodes on Kubernetes, all you need to provide in the Yugabyte Platform console is your Kubernetes provider credentials. The Yugabyte Platform will use those credentials to automatically provision and de-provision the pods that run Yugabyte.
+
+Before you install YugabyteDB on a Kubernetes cluster, perform the following:
+
+- Create a `yugabyte-platform-universe-management` service account.
+- Create a `kubeconfig` file of the earlier created service account to configure access to the Kubernetes cluster.
+
+### Service account creation
+
+This is the ServiceAccount whose secret can be used to generate a kubeconfig.
+
+**Notes**
+
+- It should not be deleted once it is being used by the platform.
```

Comment:
It should not be deleted once it is being used by the platform. --> It should not be deleted once it is in use by the platform.